cnf-certification-test/accesscontrol/resources/resources_test.go

```diff
@@ -83,3 +83,78 @@ func TestHasRequestsAndLimitsSet(t *testing.T) {
 		assert.Equal(t, tc.expectedResult, HasRequestsAndLimitsSet(tc.testContainer))
 	}
 }
+
+func TestHasExclusiveCPUsAssigned(t *testing.T) {
+	testCases := []struct {
+		testContainer  *provider.Container
+		expectedResult bool
+	}{
+		{ // Test Case #1 - Exclusive CPU pool selected
+			testContainer: &provider.Container{
+				Container: &corev1.Container{
+					Resources: corev1.ResourceRequirements{
+						Requests: corev1.ResourceList{
+							"cpu": resource.MustParse("2"),
+						},
+						Limits: corev1.ResourceList{
+							"cpu": resource.MustParse("2"),
+						},
+					},
+				},
+			},
+			expectedResult: true,
+		},
+		{ // Test Case #2 - Shared CPU pool selected (requests and limits are not equal)
+			testContainer: &provider.Container{
+				Container: &corev1.Container{
+					Resources: corev1.ResourceRequirements{
+						Requests: corev1.ResourceList{
+							"cpu": resource.MustParse("1"),
+						},
+						Limits: corev1.ResourceList{
+							"cpu": resource.MustParse("2"),
+						},
+					},
+				},
+			},
+			expectedResult: false,
+		},
+		{ // Test Case #3 - Shared CPU pool selected (requests and limits quantities are not an integer)
+			testContainer: &provider.Container{
+				Container: &corev1.Container{
+					Resources: corev1.ResourceRequirements{
+						Requests: corev1.ResourceList{
+							"cpu": resource.MustParse("512m"),
```

Comment:
Is 512m treated as a fraction?

Reply:
We can add one more test case with fraction value with limit 1.5

Reply:
Yes, but it's a strange number for a CPU fraction, I probably was thinking of memory when I wrote that number. I changed it to 500m, which is equivalent to 0.5. I added a new unit test with 1.5 as you suggested in any case.